In [9]:
import numpy as np
import astropy.units as u

Semi- major axis can be found using the following parameters :

In [10]:
k1 = (24.015)  # semi amplitude of RV in km/s
q = (0.1)  # mass ratio
i = (58.59)  #inclination of the orbital plane
P = (6.6528 * (10**4))   #orbital period in seconds

Here, K1 and P was calculated previously, whereas the q and i values were referenced from Gajdoš et al.

In [11]:
a = k1*P*(q+1)/(2*(np.pi)*q*(np.sin(i)))  * u.kilometer # unit = km
print(f"Semi major axis of the system is {a}")

Semi major axis of the system is 3138076.785486924 km


In [12]:
a_solar_units = a.to(u.R_sun) 
a_solar_units

<Quantity 4.51067527 solRad>

In terms of solar radius, Semi- major axis of the system is 4.5106753 R☉. 



This value is fairly close to the value (4.189 R☉) predicted by Gajdoš et al, considering the approximations in the RV modelling, as we averaged over velocity range instead of RV curve fitting. 